features/levering-koopsom.feature

```diff
@@ -0,0 +1,152 @@
+            },
+            "type": "perceel",
+            "aardCultuurBebouwd": {
+                "code": "11",
+                "waarde": "Wonen"
+            },
+            "kadastraleAanduiding": {
+                "kadastraleGemeente": {
+                    "code": "87",
+                    "waarde": "Beekbergen"
+                },
+                "perceelnummer": 4879,
+                "sectie": "K"
+            },
+            "_links": {
+                "self": {
+                    "href": "/kadastraalonroerendezaken/NL.IMKAD.KadastraalObject.76870487970000"
+                }
+            }
+        }
+        """
+
+
+    Scenario: Onroerendezaak (met koopsom & koopjaar & aardCultuurbebouwd) wordt opgevraagd met fields parameter met 1 van die drie velden erin
+        Als onroerendezaak gezocht word met /kadastraalonroerendezaken/NL.IMKAD.KadastraalObject.76870487970000?fields=koopsom
+        Dan worden de koopsom & koopjaar & aardCultuurBebouwd teruggegeven
+        En wordt de volgende json response teruggegeven:
+        """
+        {
+            "identificatie": "NL.IMKAD.KadastraalObject.76870487970000",
+            "koopsom": {
+                "koopsom": 185000,
+                "koopjaar": 2015
+            },
+            "type": "perceel",
+            "aardCultuurBebouwd": {
+                "code": "11",
+                "waarde": "Wonen"
+            },
+            "_links": {
+                "self": {
+                    "href": "/kadastraalonroerendezaken/NL.IMKAD.KadastraalObject.76870487970000?fields=koopsom"
+                }
+            }
+        }
+        """
+
+    Scenario: Onroerendezaak (met koopsom & koopjaar & aardCultuurbebouwd) wordt opgevraagd met fields parameter zonder 1 van die drie velden erin
+        Als onroerendezaak gezocht word met /kadastraalonroerendezaken/NL.IMKAD.KadastraalObject.76870487970000?fields=kadastraleAanduiding
+        Dan worden de koopsom & koopjaar & aardCultuurBebouwd niet teruggegeven
+        En wordt de volgende json response teruggegeven:
+        """
+        {
+            "identificatie": "NL.IMKAD.KadastraalObject.76870487970000",
+            "type": "perceel",
+            "kadastraleAanduiding": {
+                "kadastraleGemeente": {
+                    "code": "87",
+                    "waarde": "Beekbergen"
+                },
+                "perceelnummer": 4879,
+                "sectie": "K"
+            },
+            "_links": {
+                "self": {
+                    "href": "/kadastraalonroerendezaken/NL.IMKAD.KadastraalObject.76870487970000?fields=kadastraleAanduiding"
+                }
+            }
+        }
+        """
+
+    Scenario: Onroerendezaak (met alleen koopsom & aardCultuurbebouwd) wordt opgevraagd zonder fields parameter
+        Als onroerendezaak gezocht word met /kadastraalonroerendezaken/NL.IMKAD.KadastraalObject.76870488070000
+        Dan worden de koopsom & aardCultuurBebouwd niet teruggegeven
```

Comment:
Betekent dit dat het teruggeven van aardCultuurBebouwd, koopsom en koopjaar niet alleen afhangt van wat je vraagt met fields, maar ook van de waarden in de bron? En wel andersom (bij fields worden dingen toch geleverd waar je niet om gevraagd hebt, bij waarden worden dingen niet geleverd waar je wel om gevraagd hebt)?

Bijvoorbeeld 24780711870000 heeft aardCultuurBebouwd én aardCultuurOnbebouwd, indicatieMetMeerObjectenVerkregen=true, koopjaar, maar geen koopsom (ik gok omdat de koopsom betrekking heeft op meerdere objecten?). Hier wordt dus geen koopjaar geleverd, geen aardCultuurBebouwd en geen aardCultuurOnbebouwd, maar wel indicatieMetMeerObjectenVerkregen?

Reply:
Zo lijkt het bedoeld te zijn, ook door Hilde ([zie hier](https://github.com/VNG-Realisatie/Haal-Centraal-BRK-bevragen/issues/466#issuecomment-623390864)). Dit is overigens reeds geïmplementeerd, behalve dat _indicatieMetMeerObjectenVerkregen_ in jouw voorbeeld óók niet wordt geleverd.   
_( óf koopsom + aardcultuur óf geen van die velden )_